modules/custom_operations/CMakeLists.txt

```diff
@@ -4,7 +4,10 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(openvino_extensions)
+find_package(Git REQUIRED)
```

Comment:
git can be unavailable, e.g. when we are building on `conda-forge` where archive is used (with no .git folder)

can we move version extraction closer to the place where it's used?

Reply:
good point, will move it closer to cpack configuration

Reply:
excluded from this PR